functests/storagecluster_creation_test.go

```diff
@@ -0,0 +1,159 @@
+package functests_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"github.com/openshift/ocs-operator/pkg/controller/util"
+
+	deploymanager "github.com/openshift/ocs-operator/pkg/deploy-manager"
+	corev1 "k8s.io/api/core/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+)
+
+var _ = Describe("StorageCluster Creation", func() {
+	var ocsClient *rest.RESTClient
+	var parameterCodec runtime.ParameterCodec
+	var storageCluster *ocsv1.StorageCluster
+
+	BeforeEach(func() {
+		RegisterFailHandler(Fail)
+
+		deployManager, err := deploymanager.NewDeployManager()
+		Expect(err).To(BeNil())
+
+		ocsClient = deployManager.GetOcsClient()
+		parameterCodec = deployManager.GetParameterCodec()
+	})
+
+	Describe("Duplicate StorageCluster Creation", func() {
+
+		BeforeEach(func() {
+			monQuantity, err := resource.ParseQuantity("10Gi")
+			Expect(err).To(BeNil())
+
+			dataQuantity, err := resource.ParseQuantity("100Gi")
+			Expect(err).To(BeNil())
+
+			storageClassName := "gp2"
+			blockVolumeMode := k8sv1.PersistentVolumeBlock
+
+			storageCluster = &ocsv1.StorageCluster{
```

Comment:
You could export the `defaultStorageCluster()` function in `pkg/deploy-manager/storagecluster.go` and just change the name of the default StorageCluster to something else. That would help save some code dup here for the StorageCluster struct.